packages/components/src/card/body.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { BodyUI } from './styles/card-styles';
+import { useCardContext } from './context';
+
+export const defaultProps = {
+	isShady: false,
+	size: 'md',
```

Comment:
I think in other components we used `small`, `medium`... as sizes (need to be confirmed though). Should we check and consolidate if needed?

Reply:
@youknowriad Thank you! I'll make that update 🙏